.github/workflows/update-search.yml

```
@@ -47,9 +49,25 @@ jobs:
             # Commit changes
             git commit -m "Update search database from comphy-search repository [skip-deploy]"
             
-            # Rebase onto latest origin/main to avoid conflicts
-            git rebase origin/main
+            # Push with retry logic
+            MAX_RETRIES=3
+            RETRY_COUNT=0
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)